service/src/test/kotlin/no/nav/su/se/bakover/service/behandling/BehandlingTestUtils.kt

```diff
@@ -170,34 +124,6 @@ object BehandlingTestUtils {
         )
     )
 
-    internal fun createService(
```

Comment:
Denne er vel flyttet inn i en annen TestUtil-klasse for services? Kanskje vi skal se på å sammenstille disse.

Reply:
Kan gjøres adhoc